index.js

```diff
@@ -116,6 +116,27 @@ bot.on("presenceUpdate", (_, presence) => {
                 console.log("No Game detected");
             }
         }
+        if (activity.name == "Yuzu") {
+            currGame = activity.state;
+            if ( currGame && curGame != "Currently not in game" ) {
+                currGame = currGame.trim().replace(/&/g, "%26");
```

Comment:
This is where my primary worries are, I pulled it from Citra. Unlike cemu, it does not say "Currently playing (title)", so i copied and pasted this and the rest of the code from Citra